.travis/install.sh

```diff
@@ -42,7 +53,8 @@ if [ "${BUILDER}" == make ]; then
 				installmono
 			elif [ "${LANGUAGE}" == fsharp ]; then
 				installmono
-				sudo apt-get -yqq install fsharp
+        sudo apt-get -yqq install fsharp
+        installdotnetsdk
```

Comment:
indent use tab, I could fix it later if you want

Reply:
Fixed. What is going on with AppVeyor builds?